### ISSUE TITLE
Urolithiasis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7542]
**Preferred term label:**
Urolithiasis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Renal stones are formed within the kidneys, and this is called nephrolithiasis. Urolithiasis is a condition that occurs when these stones exit the renal pelvis and move into the remainder of the urinary collecting system, which includes the ureters, bladder, and urethra. 
PMID: 32644527
**Parent term (use hpo.jax.org/app)**
nephrolithiasis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 300661   PMID: 28742244  1/1 congenital onset

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
also PMID: 2227934  1/1 OMIM:614723

**Comment by pnrobinson** [comment]
added
